avocado/core/nrunner_tap.py

```diff
@@ -77,7 +77,7 @@ class RunnerApp(nrunner.BaseRunnerApp):
 
 
 def main():
-    nrunner.main(RunnerApp)
+    return nrunner.main(RunnerApp)
```

Comment:
My comments on this are better described here: https://github.com/avocado-framework/avocado/issues/3663#issuecomment-601728978